recipes/vispr/meta.yaml

```diff
@@ -9,25 +9,19 @@ source:
   sha256: dc2cf6ac9c8930b0f1f1b3a2b7f57bfae180b2ac3674060123fd2d7fe10ccd82
 
 build:
-  skip: True # [py27]
+  noarch: python
   number: 2
+  script: python -m pip install --no-deps --ignore-installed .
   entry_points:
     - vispr = vispr.cli:main
 
 requirements:
   host:
     - python
-    - setuptools
-    - numpy
-    - flask
-    - pandas >=0.17.1,<0.20
-    - pyyaml
-    - scipy
-    - scikit-learn
-    - appdirs
+    - pip
   run:
-    - python
-    - numpy
+    - python >=3
+    - numpy >=1.10
```

Comment:
@corburn, is there any reason you added those constraints?
Acc. to https://bitbucket.org/liulab/vispr/src/03dd84177d431f82943bec0b25e1540d4e674e6e/setup.py this should be Python 2 compatible (although it was `skip`ped before in the recipe) and no minimum `numpy` version is specified.

Reply:
@mbargull 

The python constraint replaced the `skip` in the recipe.
The numpy constraint resolved this error:

```
Returning non-final recipe for vispr-0.4.14-py36_2; one or more dependencies was unsatisfiable:^M                                                                                                                  
Build: None^M                                                                                                                                                                                                      
Host: The following specifications were found to be in conflict:^M                                                                                                                                                 
  - numpy=1.9^M                                                                                                                                                                                                    
  - pandas[version='>=0.17.1,<0.20'] -> numpy=1.10
```